Library/Homebrew/formula.rb

```diff
@@ -560,6 +573,25 @@ def self.path name
     Pathname.new("#{HOMEBREW_LIBRARY}/Formula/#{name.downcase}.rb")
   end
 
+  def self.search_path name
+    p = Formula.path(name)
+    return p if p.file?
+    name = name.downcase
+    taps = Dir["#{HOMEBREW_LIBRARY}/Taps/*/*/"]
+    p = taps.map { |tap| Dir["#{tap}#{name}.rb", "#{tap}Formula/#{name}.rb", "#{tap}HomebrewFormula/#{name}.rb"] }
+    p = p.flatten.map(&Pathname.method(:new)).select { |f| f.file? }
```

Comment:
Just use a regular block here, please, or use `Pathname.glob` instead of `Dir[]`